packages/koa-metrics/README.md

```diff
@@ -0,0 +1,74 @@
+# `@shopify/koa-metrics`
+
+Performance metrics implemented with DogStatsD (<https://docs.datadoghq.com/developers/dogstatsd/>) are used widely across services at Shopify. The standard metrics should be documented and following strong conventions.
+
+This package aims to provide standard middlewares and instrumentation tooling for metrics in Koa.
+
+## Installation
+
+```bash
+$ yarn add @shopify/koa-metrics
+```
+
+## Usage
+
+```javascript
+import Koa from 'koa';
+import metrics from '@shopify/koa-metrics';
+
+const app = new Koa();
+
+app.use(metrics(options));
+```
+
+### Options
+
+The passed in `options` object adheres to the following API:
+
+```javascript
```

Comment:
Don't need this part.

Reply:
I do for my editor to do proper code highlighting